src/types/global.ts

```diff
@@ -4,6 +4,12 @@ export type FilterButtonState = -1 | 0 | 1;
 
 type ZoomLevelTuple = typeof allZoomLevel;
 
+export type FocusedData = {
+  date: string;
+  resourseIndex: number;
```

Comment:
Seria bom corrigir a sintaxe aqui
```suggestion
  resourceIndex: number;
```